lib/x11utils.pm

```diff
@@ -29,6 +29,7 @@ our @EXPORT = qw(
   turn_off_plasma_screenlocker
   turn_off_gnome_screensaver
   turn_off_gnome_screensaver_for_gdm
+  turn_off_gnome_screensaver_for_running_gdm
```

Comment:
Any difference between these 2 functions? we have qemu/svirt/zvm/powerVM backend, do we cover all?

Reply:
I don't think this is related with backends, just as the different name of the functions, turn_off_gnome_screensaver_for_running_gdm is for 'running' gdm.